src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -228,6 +233,7 @@ public void writeLogTo(XMLOutput out) throws IOException {
 
     @Extension
     public static class DescriptorImpl extends TriggerDescriptor {
+        private static final Logger LOGGER = Logger.getLogger(DescriptorImpl.class.getName());
```

Comment:
I need see log from Descriptor and not from multiple triggers. Descriptor is singleton and i don't need to track from what Describable instance this log was printed.